sys/shell/commands/sc_sntp.c

```diff
@@ -44,7 +44,39 @@ int _ntpdate(int argc, char **argv)
         return 1;
     }
     sock_udp_ep_t server = { .port = NTP_PORT, .family = AF_INET6 };
-    ipv6_addr_from_str((ipv6_addr_t *)&server.addr, argv[1]);
+    ipv6_addr_t *addr = (ipv6_addr_t *)&server.addr;
```

Comment:
This looks copy&pasted from ```sc_icmpv6_echo.c```!?

Reply:
Yes, as suggested by @miri64 [above](https://github.com/RIOT-OS/RIOT/pull/8358#discussion_r161298452). 
How do you suggest to deduplicate this. i.e. where do you think the duplicated code should go? somewhere in `sys/net/network_layer/ipv6/addr/` ?.
Also, this PR is merged already - so you request a follow-up?

Reply:
I didn't realize that this was already merged. Guess you got lucky. :)